resources/js/admin/users/components/SecurityLogsModal.vue

```diff
@@ -191,7 +192,38 @@ export default {
           auxArray[this.capitalizeKey(key)] = this.booleanToString(value);
         }
       }
-      return auxArray;
+
+      auxArraySort  = this.sortModalArray(auxArray);
+
+      return auxArraySort;
+    },
+    /**
+     * Sort modal Array
+     */
+    sortModalArray(auxArray) {
+      let sortKey = ["Name"];
+      let auxArraySorted = {};
+      let specialKey = Object.keys(auxArray).find(key => ["Created_at", "Deleted_at", "Updated_at", "Last_modified", "Accessed_at"].includes(key));
```

Comment:
this is a const never changes,
change the name to dateKey

Reply:
Solved.